samples/WebApp/JavaScript.html

```diff
@@ -0,0 +1,109 @@
+<!--
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ -->
+<script>
+  /**
+   * Create a wrapped version of google.script.run that
+   * is adapted for Angular promises.
+   */
+  var ScriptService = function($q) {
+    var self = this;
+    var promisify = function(key) {
+      return function() {
+        var args = arguments;
+        return $q(function(resolve, reject) {
+          google.script.run
+            .withSuccessHandler(resolve)
+            .withFailureHandler(reject)
```

Comment:
Missing `;`?

Reply:
No, the next line is part of the same statement.